scripts/generator/graphql_generator/csharp/SDK/Android/AndroidNativeCheckout.cs.erb

```diff
@@ -16,10 +16,28 @@ namespace <%= namespace %>.SDK.Android {
         private String MerchantName;
         private String CountryCodeString;
         private AndroidPayEventReceiverBridge AndroidPayEventBridge;
+        #if !SHOPIFY_MONO_UNIT_TEST
+        private AndroidJavaObject _androidPayCheckoutSession;
+        private AndroidJavaObject AndroidPayCheckoutSession {
```

Comment:
This pattern seems a bit weird since we're using 2 private variables here. Could we not just create the session object in the constructor? Not sure what benefit we get by lazy loading it like this unless this was meant to be a public var.

Reply:
C# doesn't seem to have backing fields for properties other than its own extra property dedicated for this purpose. I just wanted to avoid the Java-like 
`GetAndroidPayCheckoutSession()` approach, but you're right, there's really no benefit in lazy loading it, I did not think much about that. Thanks for the heads up.

Reply:
https://github.com/Shopify/unity-buy-sdk/pull/360/commits/61e063a40038be9856987e8bbb47e827c3ab61e4